### PR TITLE
Drone-Agronom

### PR DESCRIPTION
**Pull request by slavikyd**

### CODE REVIEW

**Comment by okalachev** [comment]
Is this a draft or a full article?

File: docs/ru/SUMMARY.md [hunk 1]
```diff
@@ -121,7 +121,7 @@
   * [Калибровка камеры (legacy)](camera_calib.md)
   * [Управление дроном для оценки позы человека](human_pose_estimation_drone_control.md)
   * [Распознавание видов агрокультур](agriculture.md)
-
```
Comment (okalachev) [inline docs/ru/SUMMARY.md:124]: You don't have to remove this line.

File: docs/ru/drone-agronom.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Дрон-Агроном
```
Comment (okalachev) [inline docs/ru/drone-agronom.md:1]: Formatting is lacking.

Also, I suggest you to add some images, YouTube videos, etc...

**Review by okalachev — CHANGES_REQUESTED** [review]

File: docs/ru/drone-agronom.md (hunk 2 above)
Comment (slavikyd) [reply docs/ru/drone-agronom.md:1]: хорошо мы сделаем, просто пока их нету...

Comment (slavikyd) [reply docs/ru/drone-agronom.md:1]: а как делать форматирование?

**Comment by slavikyd** [comment]
> Is this a draft or the full article?

Да это драфт. Полностью сделать не успеваю т.к я сейчас на ДПП...

File: docs/ru/drone-agronom.md (hunk 2 above)
Comment (okalachev) [reply docs/ru/drone-agronom.md:1]: Смотрите другие статьи.

Comment (slavikyd) [reply docs/ru/drone-agronom.md:1]: Хорошо

File: docs/ru/SUMMARY.md (hunk 1 above)
Comment (slavikyd) [reply docs/ru/SUMMARY.md:124]: Уже исправил.

**Comment by okalachev** [comment]
I edited a little and merged your article as is for now.

You can continue working and send updates as other PR tomorrow and later. Also, please, give me some your contacts (e-mail, Telegram, ...), I will add them to the article too.

**Pull request merged**